doc/api/esm.md

```diff
@@ -283,6 +283,9 @@ const buffer = readFileSync(new URL('./data.proto', import.meta.url));
 
 > Stability: 1 - Experimental
 
+*This feature is only available with the `--experimental-import-meta-resolve`
+command flag enabled.*
```

Comment:
```suggestion
This feature is only available with the `--experimental-import-meta-resolve`
command flag enabled.
```
Should we remove the surrounding asterisks since most of the lines stating similar info don't have them?